samples/openthread/cli/README.rst

```diff
@@ -61,6 +64,8 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+To update OpenThread libraries provided by the ``nrfxlib`` with the currently built ones please invoke ``ninja -C build/ install_openthread_libraries`` after building the CLI example from source.
```

Comment:
If dependencies are specified correctly, then it is enough to just invoke:
```
ninja -C build/ install_openthread_libraries
```

No need to do a CLI build first.
If libraries are not built, then running the target will ensure they gets built and installed.
If they are already built, they will just be installed.

```suggestion
To update OpenThread libraries provided by the ``nrfxlib``, please invoke ``ninja -C build/ install_openthread_libraries``.
```

Reply:
As discussed added modified version with the `west` instead of the `ninja` command.